common/beerocks/bwl/dwpal/mon_wlan_hal_dwpal.h

```diff
@@ -59,12 +59,16 @@ class mon_wlan_hal_dwpal : public base_wlan_hal_dwpal, public mon_wlan_hal {
     bool dwpal_get_scan_params_fg(sScanCfgParams &params)
     {
         const size_t expected_result_size = sizeof(params) + NL_ATTR_HDR;
+        size_t received_result_size       = 0;
 
-        if (expected_result_size != dwpal_nl_cmd_get(m_radio_info.iface_name,
+        if (expected_result_size !=
+            (received_result_size = dwpal_nl_cmd_get(m_radio_info.iface_name,
                                                      LTQ_NL80211_VENDOR_SUBCMD_GET_SCAN_PARAMS,
-                                                     m_nl_buffer, NL_MAX_REPLY_BUFFSIZE)) {
+                                                     m_nl_buffer, NL_MAX_REPLY_BUFFSIZE))) {
             LOG(ERROR) << "LTQ_NL80211_VENDOR_SUBCMD_GET_SCAN_PARAMS failed! returned size does "
```

Comment:
No need to specify internal structure name in the error message.
A more appropriate error message would be:
```
LOG(ERROR) << "LTQ_NL80211_VENDOR_SUBCMD_GET_SCAN_PARAMS failed: expected size" 
           << expected_result_size << ", received size " << received_result_size;
```